numba/tests/test_np_functions.py

```diff
@@ -2748,6 +2768,44 @@ def test_repeat_exception(self):
                 with self.assertRaises(TypingError):
                     nbfunc(np.ones(1), rep)
 
+    def test_windowing(self):
+        def check_window(func):
+            np_pyfunc = func
+            np_nbfunc = njit(func)
+
+            for M in [0, 1, 5, 12]:
+                expected = np_pyfunc(M)
+                got = np_nbfunc(M)
+                self.assertPreciseEqual(expected, got)
+
+            with self.assertRaises(TypingError):
```

Comment:
Same comment from below RE testing exceptions applies here.

Reply:
Thanks for catching this. I think the latest commit fixes the oversight.